src/server/views/dashboard/macros.njk

```diff
@@ -53,7 +53,7 @@
   </div>
 {% endmacro %}
 
-{% macro listItemEditActionBar(list, listItem, dashboardRoutes, titleProperty, isPinned, actionButtons) %}
+{% macro listItemEditActionBar(list, listItem, dashboardRoutes, titleProperty, isPinned, actionButtons, req) %}
```

Comment:
`src/server/components/dashboard/listsItems/controllers.ts`
☝️ could you add something like `changeMessage` to the res.render function? that way we don't have to access req directly